x-pack/legacy/plugins/triggers_actions_ui/np_ready/public/application/sections/alerts_list/components/collapsed_item_actions.tsx

```diff
@@ -95,15 +91,13 @@ export const CollapsedItemActions: React.FunctionComponent<ComponentOpts> = ({
       <EuiFormRow>
         <EuiSwitch
           name="mute"
-          checked={isMuted}
-          disabled={!canSave || !isEnabled}
+          checked={item.muteAll}
+          disabled={!canSave || !item.enabled}
```

Comment:
nit. but... I find `canSave && item.enabled` clearer than `! || !`... is it just me? :)

Reply:
But the logic would be different though?

Reply:
ah, I meant with a single `!` in the beginning :)
I just find it easier, but super nit picky